### ISSUE TITLE
Tensorflow prediction gave me right answer but not in .mlmodel in iOS

### ISSUE THREAD

**Issue by younatics** [tf-coreml/tf-coreml#160]
I made my DNN with 2 hidden layers and input data type is like this. `[None, 7]` 
In my code, I input my data 
`[[  16.           -3.           18.            6.          161.99995422    89.99995422   25.52429199]... ]]`
and gave me `16` which is correct answer in my model. 

but after I initilze my model in iOS and gave `MLMultiArray `, 
```Swift
let array = try MLMultiArray(shape: [7], dataType: .double)
            array[0] = NSNumber(value: 16.0)
            array[1] = NSNumber(value: -3.0)
            array[2] = NSNumber(value: 18.0)
            array[3] = NSNumber(value: 6.0)
            array[4] = NSNumber(value: 161.9999542)
            array[5] = NSNumber(value: 89.99995422)
            array[6] = NSNumber(value: 25.52429199)

```
Softmax result gave me wrong answer. 

Do you know how to solve this problem? :(

**Comment by younatics** [comment]
I attached my frozen dnn file and coreml model. 
[dnn.zip](https://github.com/tf-coreml/tf-coreml/files/1902038/dnn.zip)

**Comment by aseemw** [comment]
Can you first check if they match when you use coremltools predict to get the output?

**Comment by aseemw** [comment]
The outputs match when I use coremltools

```
import tensorflow as tf
from tensorflow.core.framework import graph_pb2
import numpy as np
import coremltools

X = np.array([16, -3, 18, 6, 162, 90, 25.52])

X_tf = np.reshape(X, (1,7))

output = 'output/Softmax:0'

tf.reset_default_graph()
graph_def = graph_pb2.GraphDef()
with open('frozen_dnn.pb', "rb") as f:
  graph_def.ParseFromString(f.read())

with tf.Graph().as_default() as g:
  tf.import_graph_def(graph_def, name='')

# g = tf.import_graph_def(graph_def)

with tf.Session(graph=g) as sess:
  out = sess.run(output, feed_dict={'Input:0': X_tf})

out = out.flatten()

print('tf out shape:', out.shape)
print('tf out sum, max, argmax', np.sum(out), np.amax(out), np.argmax(out))

cm = coremltools.models.MLModel('dnn.mlmodel')
inp = {}
inp['Input__0'] = X
result = cm.predict(inp)['output__Softmax__0']
print('coreml out shape: ', result.shape)
print('coreml out sum, max, argmax: ', np.sum(result), np.amax(result), np.argmax(result))
```

**Comment by younatics** [comment]
Hello @aseemw , Actually I got same result when I use it in iOS when I deleted derived data in Xcode. 
It is my fault. Thanks for sharing your time!
